File: seconddegreeequation.php
<?php
	function solve_quad ($a, $b, $c)
	{
		$d = $b*$b - 4*$a*$c;
		if ($d < 0)
			return (array(0));
		else if ($d == 0)
			return (array(-($b/2/$a)));
		else
			return (array(((-$b-sqrt($d))/2/$a), ((-$b+sqrt($d))/2/$a)));
	}
?>




"""@author: github.com/isacarcanjo"""

File: seconddegreeequation.c



void second_degree_equation(float a, float b, float c);

int main()
{
    float a, b, c;
    printf("Enter a,b,c of the second degree equation ax^2+bx+c=0:\n");
    scanf("%f %f %f", &a, &b, &c);

    second_degree_equation(a, b, c);

    return 0;
}

void second_degree_equation(float a, float b, float c)
{
    float D, x, x1, x2;

    D = b*b - 4*a*c;

    if(D > 0)
    {
        x1 = (-b + sqrt(D)) / (2*a);
        x2 = (-b - sqrt(D)) / (2*a);

        printf("Equation roots are:\nx1 = %f\nx2 = %f\n", x1, x2);
    }
    else if(D == 0)
    {
        x = -b / (2*a);
        printf("Equation root is:\nx = %f\n", x);
    }
    else
    {
        printf("No real roots\n");
    }
}




"""@author: github.com/isacarcanjo"""



